[IMP] reference/actions: update doc to reflect new behaviour (#11043)
odoo/documentation: content/developer/reference/backend/actions.rst

``nextcall``
    Next planned execution date of this action (date/time format)

``priority``

>     Priority of the action when executng multiple action at the same time

Reviewer: ```suggestion
    Priority of the action when executing multiple action at the same time
```

Advanced use: Batching
----------------------

When executing a scheduled action, it's recommended to try batching progress in order
to avoid hogging a worker for a long period of time and possibly running into timeout exceptions.

Odoo provides a simple API for scheduled action batching;

.. code-block:: python

      self.env['ir.cron']._notify_progress(done=XX:int, remaining=XX:int)

This method allows the scheduler to know if progress was made and whether there is

> still remaining work must be done.

Reviewer 2: ```suggestion
still remaining work that must be done.
```

By default, if the API is used, the scheduler tries to process 10 batches in one sitting.
If there are still remaining tasks after those 10 batches, a new cron call will be executed as
soon as possible.

Advanced use: Triggers
----------------------

For more complex use cases, Odoo provides a more advanced way to trigger
scheduled actions directly from business code.

.. code-block:: python

      action_record._trigger(at=XX:date)

Security
--------

> To avoid a fair usage of resources among scheduled actions, some security measure ensure the

Reviewer 2: ```suggestion
To avoid a fair usage of resources among scheduled actions, some security measures ensure the
```

correct functioning of your scheduled actions.

> - If an scheduled action encounter an error or a timeout three consecutive times,

Reviewer 2: ```suggestion
- If a scheduled action encounter an error or a timeout three consecutive times,
```

> To avoid a fair usage of resources among scheduled actions, some security measure ensure the
> correct functioning of your scheduled actions.
> 
> - If an scheduled action err or timeout 3 consecutive times,
>   it will skip it's current execution and be considered as failed.
> - If an scheduled action fail it execution 5 times consecutively over a period of at least
>   7 days, it will be deactivated and will notify the DB admin.

Reviewer 2: ```suggestion
To avoid a fair usage of resources among scheduled actions, some security measures ensure the
correct functioning of your scheduled actions.

- If a scheduled action encounters an error or a timeout three consecutive times,
  it will skip its current execution and be considered as failed.
- If a scheduled action fails its execution five consecutive times over a period of at least
  seven days, it will be deactivated and will notify the DB admin.
```
